src/form/wr.jl

```diff
@@ -63,21 +63,21 @@ p[t_idx] ==        g*w_to[i] + (-g*tr-b*ti)/tm*(wr[i]) + (-b*tr+g*ti)/tm*(-wi[i]
 q[t_idx] == -(b+c/2)*w_to[i] - (-b*tr+g*ti)/tm*(wr[i]) + (-g*tr-b*ti)/tm*(-wi[i])
 ```
 """
-function constraint_ohms_tp_yt_to_on_off(pm::GenericPowerModel{T}, n::Int, h::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractWRForm
-    p_to = var(pm, n, h, :p, t_idx)
-    q_to = var(pm, n, h, :q, t_idx)
+function constraint_ohms_tp_yt_to_on_off(pm::GenericPowerModel{T}, n::Int, c::Int, i, f_bus, t_bus, f_idx, t_idx, g, b, g_to, b_to, tr, ti, tm, vad_min, vad_max) where T <: PMs.AbstractWRForm
+    p_to = var(pm, n, c, :p, t_idx)
+    q_to = var(pm, n, c, :q, t_idx)
     w    = var(pm, n, :w)
     wr   = var(pm, n, :wr)
     wi   = var(pm, n, :wi)
 
-    @constraint(pm.model, p_to ==  ( g_to[h]+g[h,h]) * w[(t_bus, h)] +
-                                sum( g[h,i] * wr[(t_bus, t_bus, h, i)] +
-                                     b[h,i] *-wi[(t_bus, t_bus, h, i)] for i in PMs.phase_ids(pm) if i != h) +
-                                sum(-g[h,i] * wr[(f_bus, t_bus, h, i)] +
-                                    -b[h,i] *-wi[(f_bus, t_bus, h, i)] for i in PMs.phase_ids(pm)) )
-    @constraint(pm.model, q_to == -( b_to[h]+b[h,h]) * w[(t_bus, h)] -
-                                sum( b[h,i] * wr[(t_bus, t_bus, h, i)] -
-                                     g[h,i] *-wi[(t_bus, t_bus, h, i)] for i in PMs.phase_ids(pm) if i != h) -
-                                sum(-b[h,i] * wr[(f_bus, t_bus, h, i)] +
-                                     g[h,i] *-wi[(f_bus, t_bus, h, i)] for i in PMs.phase_ids(pm)) )
+    @constraint(pm.model, p_to ==  ( g_to[c]+g[c,c]) * w[(t_bus, c)] +
+                                sum( g[c,i] * wr[(t_bus, t_bus, c, i)] +
+                                     b[c,i] *-wi[(t_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) +
+                                sum(-g[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                    -b[c,i] *-wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
+    @constraint(pm.model, q_to == -( b_to[c]+b[c,c]) * w[(t_bus, c)] -
+                                sum( b[c,i] * wr[(t_bus, t_bus, c, i)] -
+                                     g[c,i] *-wi[(t_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm) if i != c) -
+                                sum(-b[c,i] * wr[(f_bus, t_bus, c, i)] +
+                                     g[c,i] *-wi[(f_bus, t_bus, c, i)] for i in PMs.conductor_ids(pm)) )
```

Comment:
may want to change these `i` to `d` in the constraint sums as well so they don't collide with the `i` in the function arguments.